### PR TITLE
Add _get_bucket_name() method to query the bucket name

### PR DESCRIPTION
**Pull request by tiffanyn108**
Fixed: # [6974](https://github.com/red-hat-storage/ocs-ci/issues/6974)

Added _get_bucket_name() method to query the bucket name.

### CODE REVIEW

File: ocs_ci/ocs/hsbench.py [hunk 1]
```diff
@@ -42,6 +42,20 @@ def __init__(self):
         self.result = self.hsbench_cr["output_file"]
         self.hsbench_dir = mkdtemp(prefix="hsbench-")
 
+    def _get_bucket_name(self, bucket_num):
+        """
+        Get bucket name from bucket number.
+        Args:
```
Comment (petr-balogh) [inline ocs_ci/ocs/hsbench.py:48]: Please put one empty line before Args

File: ocs_ci/ocs/hsbench.py [hunk 2]
```diff
@@ -42,6 +42,20 @@ def __init__(self):
...
+            bucket_num (int): Number of bucket
+        Returns:
+            bucket_name (str): Name of bucket
```
Comment (petr-balogh) [inline ocs_ci/ocs/hsbench.py:51]:
```suggestion
            str: Name of bucket
```
Documenting return type only

**Comment by petr-balogh** [comment]
I also see DCO — DCO is not happy so you have unsigned commit ;) Please squash all of them into one and make sure it's signed ;)

**Review by petr-balogh — APPROVED** [review]

**Review by sagihirshfeld — APPROVED** [review]
LGTM

**Pull request merged**